composer.json

```diff
@@ -10,13 +10,13 @@
         }
     ],
     "require": {
-        "zendframework/zend-http": "~2.2.5",
-        "zendframework/zend-stdlib": "~2.2.5",
-        "zendframework/zend-uri": "~2.2.5"
+        "php": ">=5.5.0",
+        "zendframework/zend-stdlib": "~2.2.5"
     },
     "require-dev": {
         "phpunit/PHPUnit": "3.7.*",
         "satooshi/php-coveralls": ">=0.6.0",
+        "psr/http-message": "1.0.*@dev",
```

Comment:
Also, without a explicit composer dependency you are simple relying in a interface name, not in a specific contract, it makes developers able to declare their own interface named `Psr\Http\Message\MessageInterface` with a incompatible API.